cmd/graph.go

```diff
@@ -47,6 +48,12 @@ var graphCmd = &cobra.Command{
 	Long:  "generate topology graph based on the topology definition file and running containers\nreference: https://containerlab.srlinux.dev/cmd/graph/",
 
 	RunE: func(cmd *cobra.Command, args []string) error {
+
```

Comment:
I wonder why by adding `WithTopoFile` clab option we haven't stoped the graphing if the file is not present?
This option should have checked if the file is present